packages/react-components/react-motion-components-preview/library/src/components/Fade/Fade.test.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Fade, FadeExaggerated, FadeSnappy } from './Fade';
+import { render, waitFor } from '@testing-library/react';
+import { motionTokens } from '@fluentui/react-motion';
+import { mockAnimation } from '../../testing/testUtils';
+
+describe('Fade motion component', () => {
+  let animateSpy: jest.SpyInstance;
+  const testElement = <div data-testid="fade-content">Test</div>;
+
+  beforeAll(() => {
+    HTMLElement.prototype.animate = () => mockAnimation;
```

Comment:
See suggestion on `mockAnimation` to make this a function call that returns a new mock Animation object each time:
```suggestion
    HTMLElement.prototype.animate = () => mockAnimation();
```